kobo/apps/superuser_stats/admin.py

```diff
@@ -48,6 +57,79 @@ def queryset(self, request, queryset):
         return queryset.filter(**condition)
 
 
+class CountryFilter(admin.SimpleListFilter):
+    title = 'Country'
+    parameter_name = 'Country'
+
+    def __init__(self, request, params, model, model_admin):
+        super().__init__(request, params, model, model_admin)
+        self.__model = model
+
+    def lookups(self, request, model_admin):
+        return COUNTRIES
+
+    def queryset(self, request, queryset):
+        if not self.value():
+            return queryset
+
+        country = self.value()
+        return queryset.filter(settings__country__value=country)
+
+
+class SubmissionsByCountry(admin.ModelAdmin):
+    change_list_template = 'submissions_by_country.html'
+    list_filter = (('date_created', DateFieldListFilter), CountryFilter)
+    actions = None
+
+    def changelist_view(self, request, extra_context=None):
+        response = super().changelist_view(
+            request,
+            extra_context=extra_context,
+        )
+        response.context_data['summary'] = self.__get_serialized_data(request)
+        return response
+
+    def __get_serialized_data(self, request) -> list:
+        cl = self.get_changelist_instance(request)
+        qs = cl.get_queryset(request)
+
+        data = []
+
+        asset_filter = CountryFilter(
+            request, request.GET.dict(), Asset, self.__class__
+        )
+
+        asset_queryset = Asset.objects.filter(
+            asset_type=ASSET_TYPE_SURVEY
+        )
+
+        records = asset_filter.queryset(request, asset_queryset).order_by()
+
+        # Filter for individual countries
+        for code, name in COUNTRIES:
+            assets = records.filter(
+                settings__country__value=code,
+            )
+            if assets.count() != 0:
+                xform_id_strings = list(assets.values_list(
+                    '_deployment_data__backend_response__id_string', flat=True
+                ).filter(
+                    _deployment_data__backend='kobocat',
+                    asset_type=ASSET_TYPE_SURVEY,
+                ))
+
+                instances_count = ReadOnlyKobocatInstance.objects.filter(
+                    xform__id_string__in=xform_id_strings,
+                ).count()
```

Comment:
We may be able to speed-up things a little bit. The number of submissions is already calculated within the `logger_xform` table. 

What about returning the count directly from  `logger_xform` with `SUM()`? I
```python
result = KobocatXForm.objects.filter(xform__id_string__in=xform_id_strings).aggregate(instances_count=Sum('num_of_submissions'))
instances_count = result['instances_count']
```